bodo/src/test/java/org/apache/calcite/test/BodoSqlToRelConverterTest.java

```diff
@@ -99,4 +111,58 @@ public class BodoSqlToRelConverterTest extends SqlToRelTestBase {
         "on value.value = dept2.value";
     sql(sql).ok();
   }
+
+  @Test void testCreateTableOrderBy() {
+    // Tests an error specific to CREATE TABLE with "WITH" and "ORDER BY" clauses
+    // found while doing a test run on tpch Q2
+    final String sql = "CREATE TABLE keaton_testing AS ("
```

Comment:
rename this to something not `keaton_testing` lol

Reply:
Be more specific with this comment. What error is it? 

Also delete comment about "found while doing a test on tpch Q2", put that comment in the PR itself

Reply:
Done and done.